library/src/main/java/eightbitlab/com/blurview/BlurView.java

```diff
@@ -77,16 +81,48 @@ protected void onAttachedToWindow() {
         }
     }
 
+    /**
+     * @see #setupWith(ViewGroup, BlurAlgorithm, float)
+     */
+    public BlurViewFacade setupWith(@NonNull ViewGroup rootView) {
+        return this.setupWith(rootView, BlurController.DEFAULT_SCALE_FACTOR);
+    }
+
+    /**
+     * @see #setupWith(ViewGroup, BlurAlgorithm, float)
+     */
+    @SuppressWarnings("deprecation")
+    public BlurViewFacade setupWith(@NonNull ViewGroup rootView, float scaleFactor) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
```

Comment:
I wouldn't want to do that.
Having an explicit choice and being able to provide own implementation of `BlurAlgorithm` in`setupWith` is more important than saving a couple of lines of code on the call side